common/azure-common-test/src/main/java/com/azure/common/test/namer/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package contains classes of resource random namer.
+ */
+package com.azure.common.test.namer;
```

Comment:
I think the old utils package was sufficient - we don't need a namer package in my opinion.

Reply:
Will move back to util.